<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WizardPageJV</class>
 <widget class="QWizardPage" name="WizardPageJV">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <property name="title">
   <string>Page 2</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Support Pysique</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="QRadioButton" name="w_supportSteam">
        <property name="text">
         <string>Steam</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="w_supportCD">
        <property name="text">
         <string>CD</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QRadioButton" name="w_supportNon">
        <property name="text">
         <string>Non</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QRadioButton" name="w_supportSteamCD">
        <property name="text">
         <string>Steam et CD </string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Support</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Type</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Editeur</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Studio</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="lineEdit_4"/>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="lineEdit_3"/>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="w_extension">
     <property name="toolTip">
      <string>Unchecked: jeu de base, Partially check: Extension, Check: Stand alone</string>
     </property>
     <property name="text">
      <string>Standalone/Extension</string>
     </property>
     <property name="tristate">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="comboBox">
     <item>
      <property name="text">
       <string>PC</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>GameBoy</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Nintendo DS</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PlayStation</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PlayStation 2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>XBox 360</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>IPhone</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Android</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>w_extension</tabstop>
  <tabstop>w_supportCD</tabstop>
  <tabstop>w_supportNon</tabstop>
  <tabstop>w_supportSteam</tabstop>
  <tabstop>w_supportSteamCD</tabstop>
  <tabstop>lineEdit_3</tabstop>
  <tabstop>lineEdit_4</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
